app/index.html

```diff
@@ -18,7 +18,13 @@
                     <h2><i class="fa fa-newspaper-o"></i>Recent News</h2>
                 </div>
                 <ul>
-                    <li><div class="container-news"></div></li>
+                  <li>
+                    <div class="container-news">
+                      <div class="loading-dots">
+                        <h1 class="dot one">.</h1><h1 class="dot two">.</h1><h1 class="dot three">.</h1>
```

Comment:
I don't remember having these as `h1` elements, but I guess I did? `span`s would be more fitting.